tests/board_test.py

```diff
@@ -0,0 +1,66 @@
+from tic_tac_toe.board import Board
+
+
+def test_tic_tac_toe_initialized_with_a_3_x_3_board():
+    new_board = Board()
+
+    assert len(new_board.board) == 9
+
+
+def test_tic_tac_toe_move():
+    new_board = Board()
+
+    new_board.move(selection=3)
+
+    assert new_board.board[2] == "X"
+
+
+def test_turn_count_keeps_track_of_number_of_turns():
+    new_board = Board()
+
+    new_board.board = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+    assert new_board.turn_count() == 0
+
+    new_board.board = ["X", " ", " ", " ", " ", " ", " ", " ", " "]
+
+    assert new_board.turn_count() == 1
+
+    new_board.board = ["X", "O", " ", " ", " ", " ", " ", " ", " "]
+
+    assert new_board.turn_count() == 2
```

Comment:
As a rule of thumb, tests should have a single assertion. More than one assertion suggests the test is testing more than one thing. Here you are in the gray zone of asserting more than one instance of a behavior, but not every possible behavior. You might want to consider refactoring this sort of test with a feature in pytest called `parametrize`.
https://docs.pytest.org/en/latest/parametrize.html